src/explore-education-statistics-frontend/src/prototypes/components/PrototypeSearchForm.tsx

```diff
@@ -138,6 +134,80 @@ class PrototypeSearchForm extends Component<{}, State> {
     this.setState({ searchResults });
   }
 
+  private static openTab(target: Element, container: Element) {
+    // find the panel in which the element lives
+    const panel = PrototypeSearchForm.parentUntilClassname(
+      target,
+      'govuk-tabs__panel',
+    );
+
+    // find all the sections within the container
+    const allSections = container.querySelectorAll('.govuk-tabs__panel');
+
+    // get the index of the panel within the container
+    const selectedPanel = Array.from(allSections).indexOf(panel);
+
+    // only proceed if all this is valid
+    if (selectedPanel >= 0) {
+      // find the selected anchor
+      const anchor = container
+        .querySelectorAll('ul.govuk-tabs__list > li.govuk-tabs__list-item')
+        .item(selectedPanel)
+        .querySelector('a');
+
+      if (anchor) {
+        // Deselect all the selected tabs, make sure we found the anchor first
+        container
+          .querySelectorAll(
+            'ul.govuk-tabs__list > li.govuk-tabs__list-item a.govuk-tabs__tab--selected',
+          )
+          .forEach(_ => {
```

Comment:
Rename this `el` or something similar.

I think Lodash is also being imported in as `_` which confuses things.

I would also suggest that we shouldn't be importing Lodash as `_` anyway as this import implicitly causes the entirety of Lodash to be imported (causing bloat). We should use individual imports on the required functions e.g:

```javascript
import intersection from lodash/intersection
```